app/controllers/facility_notifications_controller.rb

```diff
@@ -107,7 +107,7 @@ def mark_as_reviewed
   def notify_accounts
     @accounts_to_notify.each do |account, facility|
       account.notify_users.each do |u|
-        Notifier.review_orders(:user => u, :facility => facility, :account => account).deliver
+        Notifier.delay.review_orders(:user => u, :facility => facility, :account => account)
```

Comment:
When it takes basic ActiveRecord objects, it seems to be fine. I am going to change the one mailer that we send `self` back (since that one's already a background job anyways).